sig-apps/Charter.md

```diff
@@ -0,0 +1,141 @@
+# SIG Apps Charter
+
+## Scope
+
+SIG Apps Covers deploying and operating applications in Kubernetes with a focus on the application developer and application operator experience.
+
+This includes but is not limited to:
```

Comment:
I can see where @r2d4 is coming from but I also fail to understand what exactly is wrong with this wording. Methinks it would help tremendously if a concrete alternative wording would be PRed here rather than just pointing out that we have scope creep with the current wording. Don't get me wrong, I also do worry about scope creep, however I can't think of a crisper way to state it.

Reply:
The reason behind this language is the scope is:

> SIG Apps Covers deploying and operating applications in Kubernetes with a focus on the application developer and application operator experience.

What if something comes up that's not on the list but within the broader scope definition? It would be easy to start niting due to the list. I'll update the language to be friendlier.